easybuild/easyconfigs/l/LCov/LCov-1.13-GCCcore-7.2.0.eb

```diff
@@ -0,0 +1,28 @@
+easyblock = 'ConfigureMake'
+
+name = 'LCov'
+version = '1.13'
+
+homepage = 'http://ltp.sourceforge.net/coverage/lcov.php'
+description = "LCOV - the LTP GCOV extension"
+
+toolchain = {'name': 'GCCcore', 'version': '7.2.0'}
+
+source_urls = ["http://downloads.sourceforge.net/ltp/"]
```

Comment:
@rubendibattista Please use `SOURCEFORGE_SOURCE` here (which uses `https` now thanks to the changes in https://github.com/easybuilders/easybuild-framework/pull/2648)